BigQuery/tests/Unit/ModelTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\BigQuery\Tests\Unit;
+
+use Google\Cloud\BigQuery\Connection\ConnectionInterface;
+use Google\Cloud\BigQuery\Model;
+use Google\Cloud\Core\Exception\NotFoundException;
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+
+/**
+ * @group bigquery
+ */
+class ModelTest extends TestCase
+{
+    public $connection;
+    public $projectId = 'myProjectId';
```

Comment:
Same note here as above, also don't mind making the connection private like done in the other test. There was no good reason for them to be public in the original tests you based this off of :).